ui/raidboss/data/05-shb/raid/e12s.js

```diff
@@ -312,124 +312,241 @@ export default {
   timelineReplace: [
     {
       'locale': 'de',
-      'missingTranslations': true,
       'replaceSync': {
         'Beastly Sculpture': 'Abbild eines Löwen',
         'Bomb Boulder': 'Bomber-Brocken',
         'Chiseled Sculpture': 'Abbild eines Mannes',
         'Eden\'s Promise': 'Edens Verheißung',
         'Guardian Of Eden': 'Wächter von Eden',
         'Ice Pillar': 'Eissäule',
+        'Oracle Of Darkness': 'Orakel der Dunkelheit',
+        'Oracle of Darkness': 'Orakel der Dunkelheit',
```

Comment:
Hello,
Is there a reason for this line to appear twice ?

Reply:
the reason is, that i am blind xD

Reply:
ok nevermind th "o" in "of" is once capitalized and once not. Cant remember if sync was case sensitive. Will clarify this with quis and adjust acordingly